docs/Formula-Cookbook.md

```diff
@@ -691,7 +691,7 @@ The symlinks created by [`install_symlink`](https://www.rubydoc.info/github/Home
 
 ### Handling files that should persist over formula upgrades
 
-For example, Ruby 1.9’s gems should be installed to `var/lib/ruby/` so that gems don’t need to be reinstalled when upgrading Ruby. You can usually do this with symlink trickery, or *better* a configure option.
+For example, Ruby 1.9’s gems should be installed to `var/lib/ruby/` so that gems don’t need to be reinstalled when upgrading Ruby. You can usually do this with symlink trickery, or (ideally) a configure option.
```

Comment:
You can usually do this with symlink trickery, or (ideally) a configure option.

could be

    You can usually do this with a configure option, or symlink trickery, alternatively.

Placing the preferred one first and then saying the other is an alternative will lead people to the first in most cases.